api/models/points.js

```diff
@@ -1,32 +1,49 @@
 const mongoose = require('mongoose');
 const { composeMongoose } = require('graphql-compose-mongoose');
-
+let p = Math.floor(Math.random() * (50 - 25) + 25);
```

Comment:
use `const` since we initialize this variable and dont change it
```suggestion
const p = Math.floor(Math.random() * (50 - 25) + 25);
```

consider renaming `p` to something more descriptive, e.g. `defaultPoints` and add a comment for why `p` is calculated this way

Reply:
my thing about using `let` vs `const` was that the value of `p` changes each time it instantiates, but yeah. looking at this

Reply:
if you want it to consistently change consider making a helper function